contracts/utils/contracts/src/LibAddress.sol

```diff
@@ -1,6 +1,6 @@
 /*
 
-  Copyright 2018 ZeroEx Intl.
+  Copyright 2019 ZeroEx Intl.
```

Comment:
Lol, I thought I was the only one doing this.

Reply:
Nope lol, I try to do it in every file I touch